host/4/dotnet-build.yml

```diff
@@ -77,7 +77,7 @@ jobs:
           dockerImagePath: "dotnet-isolated"
           dockerImageName: "-dotnet-isolated7.0-appservice"
         _isolated-net-7-mariner:
-          dotnetVersion: "dotnet7-isolated"
+          dotnetVersion: "dotnet7-isolated-mariner"
```

Comment:
Good catch on this.  Both images are being tested using mariner now and succeed at those tests?

Reply:
Also please check the rest of the repo to make sure nothing else is affected by the dotnet7 mariner image name change.  You shouldn't need to change any other pipelines but please double check. 

For example, make sure that the image produced by the build pipeline has the same exact name.  This is important to maintain release pipelines. If the image name changes it could break a downstream pipeline

Reply:
Yes, added links in the description for the successful builds. Image names have not been affected by this change, only the display name for the tests.